extension/src/lib/chromeSettings.js

```diff
@@ -0,0 +1,170 @@
+    },
+    defaultSettings: {
+        // not all of the following settings are in the common terms yet.
+        // and may need to be updated once they are added there.
+        fontSize: 1, // from fontSize
+        lineSpace: 1, // from lineSpace
+        selectionTheme: "default", // from highlightColor
+        contrastTheme: "default", // from highContrastEnabled and highContrastTheme
+        characterSpace: 1, // from characterSpace
+        inputsLargerEnabled: false, // from inputsLargerEnabled
+        selfVoicingEnabled: false, // from selfVoicingEnabled
+        tableOfContentsEnabled: false, // from tableOfContents
+        dictionaryEnabled: false, // from supportTool
+        simplifiedUiEnabled: false, // from simplifiedUiEnabled
+        syllabificationEnabled: false // from syllabificationEnabled
+    },
+    components: {
+        click2Speech: {
+            type: "gpii.chrome.extensionHolder",
+            createOnEvent: "onLoadExtensionHolders",
+            options: {
+                settingName: "text-to-speech",
+                extensionId: "djfpbemmcokhlllnafdmomgecdlicfhj",
+                name: "click2speech",
+                installationUrl: "https://chrome.google.com/webstore/detail/click2speech/djfpbemmcokhlllnafdmomgecdlicfhj",
+                model: {
+                    extensionEnabled: "{settings}.model.settings.selfVoicingEnabled"
+                }
+            }
+        },
+        dictionary: {
+            type: "gpii.chrome.extensionHolder",
+            createOnEvent: "onLoadExtensionHolders",
+            options: {
+                settingName: "dictionary",
+                extensionId: "mgijmajocgfcbeboacabfgobmjgjcoja",
+                name: "Google Dictionary (by Google)",
+                installationUrl: "https://chrome.google.com/webstore/detail/google-dictionary-by-goog/mgijmajocgfcbeboacabfgobmjgjcoja",
+                model: {
+                    extensionEnabled: "{settings}.model.settings.dictionaryEnabled"
+                }
+            }
+        },
+        domSettingsApplier: {
+            type: "gpii.chrome.domSettingsApplier",
+            options: {
+                model: "{settings}.model"
+            }
+        },
+        zoom: {
+            type: "gpii.chrome.zoom",
+            options: {
+                model: {
+                    magnifierEnabled: true, // set to true because fontSize is always enabled
+                    magnification: "{settings}.model.settings.fontSize"
+                }
+            }
+        },
+        wsConnector: {
+            type: "gpii.wsConnector",
+            options: {
+                solutionId: "net.gpii.uioPlus",
+                flowManager: "ws://localhost:8081/browserChannel",
+                retryTime: 10
+            }
+        }
+    },
+    dynamicComponents: {
+        notification: {
+            type: "gpii.chrome.notification",
+            createOnEvent: "onNotifyOfMissingExtension",
```

Comment:
This event has a slightly crazy name. It should be named something "onCreateNotification" : P